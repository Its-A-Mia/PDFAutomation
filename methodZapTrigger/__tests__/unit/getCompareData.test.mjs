import { getCompareData } from '../../index.mjs';

// requires AWS credentials

const snapshot = {
  count: expect.any(Number),
  value: expect.any(Array),
};

describe('getCompareData function', () => {
  it('grabs compare data from a .json file within an S3 bucket', async () => {
    const data = await getCompareData();
    expect(data).toMatchSnapshot(snapshot);
  });
});
