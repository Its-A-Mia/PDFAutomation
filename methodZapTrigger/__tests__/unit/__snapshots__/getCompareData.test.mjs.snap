// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getCompareData function grabs compare data from a .json file within an S3 bucket 1`] = `
{
  "count": Any<Number>,
  "value": Any<Array>,
}
`;
